frontend/benefit/applicant/src/components/header/Header.tsx

```diff
@@ -14,7 +14,7 @@ import { $CustomMessagesActions } from './Header.sc';
 import { useHeader } from './useHeader';
 
 const Header: React.FC = () => {
-  const { t, locale, languageOptions, hasMessenger, handleLanguageChange } =
```

Comment:
useLocale is used inside `shared/Header` component so no need to pass it anymore.